docs/using.rst

```diff
@@ -188,6 +190,35 @@ expected output is::
 .. _README: ../components/centos/centos-k8s-singlenode-setup/README.rst
 .. _Vagrantfile: ../components/centos/centos-k8s-singlenode-setup/Vagrantfile
 
+Vagrant bi-directional folder sync
+==================================
+
+For basic usage please refer `Vagrant documentation. <https://www.vagrantup.com/docs/synced-folders/basic_usage.html>`_
+
+Vagrant bi-directional sync folder is a very useful feature in Vagrant as it is the simplest way to move files (e.g. code) between
+the workstation and the Vagrant guest.
+
+Here are options for bi-directional sync works out of box with ADB Vagrant box.
+
+* `vagrant-sshfs <https://github.com/dustymabe/vagrant-sshfs>`_ : It works with Linux/GNU, OS X and Microsoft Windows.
```

Comment:
`vagrant-sshfs <https://github.com/dustymabe/vagrant-sshfs>`_ : works with Linux/GNU, OS X and Microsoft Windows.